duqtools/submit.py

```diff
@@ -1,13 +1,40 @@
 from pydantic import BaseModel
-import duqtools.config as cfg
+import subprocess
+import duqtools.config
+from logging import debug, error, info
+from os import scandir, path
+from pathlib import Path
 
 class Submit_config(BaseModel):
-  time               :  str
-  nodes              : int
-  processors         : int
-  processors_per_case: int
+  submit_script_name : str = ".llcmd"
+  status_file        : str = "jetto.status"
 
 def submit():
-  if not cfg.config.submit:
+  cfg = duqtools.config.Config()
+  if not cfg.submit:
     raise Exception("submit field required in config file")
-  print(cfg.config.submit)
+  debug("Submit config: %s"%cfg.submit)
+
+  run_dirs = [Path(entry) for entry in scandir(cfg.workspace) if entry.is_dir()]
+  debug("Case directories: %s"%run_dirs)
+
+  for run_dir in run_dirs:
+    submission_script = Path(path.join(run_dir,cfg.submit.submit_script_name))
+    if submission_script.is_file():
+      info("Found submission script: %s ; Ready for submission"%submission_script)
+    else:
+      debug("Did not found submission script %s ; skipping this directory..."%submission_script)
+      continue 
+
+    status_file = Path(path.join(run_dir,cfg.submit.status_file))
```

Comment:
This also works: 

```suggestion
    status_file = Path(run_dir,cfg.submit.status_file)
```

Reply:
Changed it to `run_dir/cfg.submit.status_file`